include/swift/SIL/TypeLowering.h

```diff
@@ -96,35 +96,120 @@ enum IsDependent_t : unsigned {
   IsNotDependent = false,
   IsDependent = true
 };
-  
+
+/// Is a lowered SIL type trivial?  That is, are copies ultimately more
+/// than just bit-copies, and it does it take any work to destroy?
```

Comment:
Nitpick: The second sentence describes conditions for being non-trivial, not trivial.